website/docs/r/launch_template.html.markdown

```diff
@@ -10,16 +10,24 @@ description: |-
 
 Provides an EC2 launch template resource. Can be used to create instances or auto scaling groups.
 
--> **Note:** All arguments are optional except for either `name`, or `name_prefix`.
-
 ## Example Usage
 
 ```hcl
 resource "aws_launch_template" "foo" {
   name = "foo"
 
   block_device_mappings {
-    device_name = "test"
+    # to change the type or size of the root volume, override the ami's root device name
```

Comment:
We should add this helper information to the `block_device_mappings` documentation below.  I'll adjust on merge 👍  More importantly its better to get the updated `device_name` information in the example.